projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,37 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: block;
+    width: 1rem;
+    height: 1rem;
+    border-radius: 100%;
+    box-shadow: 0 0 0.4375rem rgba(0, 0, 0, 0.3);
+    border-color: var(--tui-base-02);
+    color: var(--tui-primary);
+    box-sizing: border-box;
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0.125rem;
```

Comment:
I think it peaks out a bit more now than it should:
![image](https://github.com/taiga-family/taiga-ui/assets/11832552/aa373a35-3f44-4aca-86d0-d947197eb2ad)

Reply:
fixed